aiocache/backends/valkey.py

```diff
@@ -167,21 +210,19 @@ class RedisCache(RedisBackend):
         the backend. Default is an empty string, "".
     :param timeout: int or float in seconds specifying maximum timeout for the operations to last.
         By default its 5.
-    :param client: redis.Redis which is an active client for working with redis
+    :param client: glide.GlideClient which is an active client for working with valkey
     """
 
-    NAME = "redis"
+    NAME = "valkey"
 
     def __init__(
         self,
-        client: redis.Redis,
         serializer: Optional["BaseSerializer"] = None,
         namespace: str = "",
         key_builder: Callable[[str, str], str] = lambda k, ns: f"{ns}:{k}" if ns else k,
         **kwargs: Any,
```

Comment:
Probably want to fix this. This is why the failing test wasn't caught by mypy. We could use a TypedDict or something here.

Reply:
I'll do it in a separate PR.